package.json

```diff
@@ -0,0 +1,19 @@
+ 
```

Comment:
is this file needed? should it be in the root folder?

Reply:
This file is useful. When the parent application declares `file:../deps/membrane_sfu` in its `package.json` file, it is pointing to this file in the root folder of the library. This file points to the compiled version of the membrane_sfu javascript, so integrators who do not use typescript can load the library without changing their compilation pipeline.